packages/hash/api/src/graphql/resolvers/middlewares/loggedInAndSignedUp.ts

```diff
@@ -8,4 +8,13 @@ export const loggedInAndSignedUp: ResolverMiddleware<
   any,
   LoggedInGraphQLContext
 > = (next) => (obj: any, args: any, ctx: GraphQLContext, info: any) =>
-  loggedIn(signedUp(next))(obj, args, ctx, info);
+  // loggedIn(signedUp(next))(obj, args, ctx, info);
+  next(
+    obj,
+    args,
+    {
+      ...ctx,
+      userModel: { entityId: "00000000-0000-0000-0000-000000000000" },
+    } as any as LoggedInGraphQLContext,
+    info,
+  );
```

Comment:
Did this accidentally get committed?

Reply:
Yes, my bad, must have slipped in as I rebased

Reply:
Undone in a94a5695fbd4fa5b77731d5c57609e717876770c